types/chart.js/index.d.ts

```diff
@@ -509,6 +509,24 @@ declare namespace Chart {
         stepSize?: number;
         suggestedMax?: number;
         suggestedMin?: number;
+        minor?: MinorTickOptions;
```

Comment:
Based on the implementation it looks like `ticks.minor` and `ticks.major` actually have all the fields that `ticks` can have, except for `minor` and `major`. (which is hinted in the documentation: "Omitted options are inherited from ticks configuration.")

Also it looks like the value `false` is also supported, even if not documented.

https://github.com/chartjs/Chart.js/blob/master/src/core/core.scale.js#L148

Looks like we could implement this using [Typescript 2.8 conditional types](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-8.html#predefined-conditional-types), if we want to upgrade the typescript version from 2.3:

```
interface TickOptions { 
...
    minor?: NestedTickOptions;
    major?: NestedTickOptions;
}

type NestedTickOptions = Pick<TickOptions, Exclude<keyof TickOptions, "minor" | "major">> | false;
```

Reply:
Good catch. If we want to avoid the typescript update, what about doing this:

```
   interface TickOptions extends NestedTickOptions {
        major?: NestedTickOptions | false;
        minor?: NestedTickOptions | false;
    }

    interface NestedTickOptions {
        autoSkip?: boolean;
        ...
    }
```

Reply:
yes, that seems a good solution if we want to keep typescript 2.3 support